index.html

```diff
@@ -394,49 +395,49 @@ <h4>LET'S CONNECT</h4>
       <div class="overlay">
         <!-- <img src="images/contactformshapes.svg" alt="footershapes" /> -->
 
-        <div class="content"> 
-        <article class="footer-text">
-          <h2>Contact me</h2>
-          <p>
-            If you have an application you are interested in developing, a
-            feature that you need built or a project that needs coding. I’d love
-            to help with it
-          </p>
-        </article>
+        <div class="content">
+          <article class="footer-text">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </article>
+
+          <form action="https://formspree.io/f/xpzbopev" method="post">
+            <div class="name">
+              <input
+                type="text"
+                name="text"
+                required
+                placeholder="Yeremias"
+                maxlength="30"
+              />
+            </div>
 
-        <form action="https://formspree.io/f/xpzbopev" method="post">
-          <div class="name">
-            <input
-              type="text"
+            <div class="email">
+              <input
+                type="email"
+                name="email"
+                required
+                placeholder="natajayanj14@gmail.com"
+              />
+            </div>
+            <textarea
```

Comment:
- [ ] Please add `<label>` for all form input controls. It helps for accessibility, e.g. screen-readers.
As in your design, there's no `label`, You can use hidden attributes for your labels so it will not display but will be available for accessibility.